fivetran/framework/resources/connector.go

```diff
@@ -309,6 +309,11 @@ func (r *connector) Update(ctx context.Context, req resource.UpdateRequest, resp
 		}
 	}
 
+	/* for cases when customer missed these params */
+	plan.RunSetupTests = types.BoolValue(runSetupTestsPlan)
+	plan.TrustCertificates = types.BoolValue(trustCertificatesPlan)
+	plan.TrustFingerprints = types.BoolValue(trustFingerprintsPlan)
```

Comment:
We should take values from state here directly. If values are not set - they sshould be null. Earlier we take this value from plan with "default" value.
```
runSetupTestsPlan := core.GetBoolOrDefault(plan.RunSetupTests, false)
```
But in state it should not be updated, if it was null - it should be just types.BoolNull(), 
I'd suggest here to
```suggestion
	if plan.RunSetupTests.IsUnknown() {
		plan.RunSetupTests = state.RunSetupTests
	}
	if plan.TrustCertificates.IsUnknown() {
		plan.TrustCertificates = state.TrustCertificates
	}
	if plan.TrustFingerprints.IsUnknown() {
		plan.TrustFingerprints = state.TrustFingerprints
	}
```